src/pages/ResultsPage.js

```diff
@@ -62,28 +70,30 @@ to {
   opacity: 1;
 }
 `
-const Test = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: space-around;
-`
-
-const Animation = styled.div`
+const Wrapper = styled.section`
+  background-image: url(${Background});
+  background-size: cover;
+  background-position: center;
+  height: 100vh;
   animation-duration: 1.5s;
   animation-name: ${fadein};
+  background-color: var(--color-primary-alpha);
 `
-
-const SpinnerWrapper = styled.section`
+const Top = styled.div`
   display: flex;
-  justify-content: center;
-  padding-top: 200px;
+  align-items: center;
+  justify-content: space-around;
+  border-bottom: 1px solid var(--color-secondary);
+  background-color: var(--color-primary-alpha);
 `
 
 const BackButton = styled(Button)`
```

Comment:
i think you use this in another file as well. think about making it a component. maybe even as a link 😉